src/Sylius/Bundle/UserBundle/Form/Type/CustomerRegistrationType.php

```diff
@@ -51,28 +35,8 @@ public function buildForm(FormBuilderInterface $builder, array $options = array(
             ->add('lastName', 'text', array(
                 'label' => 'sylius.form.customer.last_name',
             ))
-            ->add('email', 'email', array(
-                'label' => 'sylius.form.customer.email',
-            ))
-            ->add('user', 'sylius_user_registration', array(
-                'label' => false,
-            ))
-            ->addEventSubscriber(new CustomerRegistrationFormListener($this->customerRepository))
-            ->addEventSubscriber(new UserRegistrationFormListener())
-            ->setDataLocked(false)
         ;
-    }
-
-    /**
-     * {@inheritdoc}
-     */
-    public function setDefaultOptions(OptionsResolverInterface $resolver)
-    {
-        $resolver->setDefaults(array(
-            'data_class' => $this->dataClass,
-            'validation_groups' => $this->validationGroups,
-            'cascade_validation' => true,
-        ));
+        parent::buildForm($builder, $options);
```

Comment:
Wanted to remain the fields order, but it should be view responisibility to render it in correct order, so fixing.